app/components/viral/pagy/limit_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="inline-flex items-center text-slate-500 dark:text-slate-400 space-x-2 mt-4">
+  <span><%= t(".items", :items => @item.pluralize(@pagy.count)) %></span>
```

Comment:
This translation can be changed to `t(".items", items: @item)`

Reply:
Thanks for seeing this, yeah, I was using it incorrectly.  While looving at it agin though, I realized that I want it in the plural form always since it is referring to the pagination size.  E.g. `Sample per page` is wrong, it should always be `Samples per page`

Reply:
Updated in e3d2f6e

Reply:
Yeah I definitely dropped the ball on the text there :facepalm: